scripts/collector/mysql/sql/data_types.sql

```diff
@@ -16,12 +16,12 @@ with src as (
         i.TABLE_NAME,
         i.DATA_TYPE
 )
-select concat(char(39), @DMA_MANUAL_ID, char(39)) as PKEY,
-    concat(char(39), @DMA_SOURCE_ID, char(39)) as DMA_SOURCE_ID,
-    concat(char(39), @DMA_MANUAL_ID, char(39)) as DMA_MANUAL_ID,
-    src.table_catalog,
-    src.table_schema,
-    src.table_name,
-    src.data_type,
-    src.data_type_count
+select concat(char(39), @PKEY, char(39)) as pkey,
+    concat(char(39), @DMA_SOURCE_ID, char(39)) as dma_source_id,
+    concat(char(39), @DMA_MANUAL_ID, char(39)) as dma_manual_id,
+    concat(char(39), src.table_catalog, char(39)) as table_catalog,
+    concat(char(39), src.table_schema, char(39)) as table_schema,
+    concat(char(39), src.table_name, char(39)) as table_name,
+    concat(char(39), src.data_type, char(39)) as data_type,
+    concat(char(39), src.data_type_count, char(39)) as data_type_count
```

Comment:
I think this converts it to some hexa because the value needs to be casted first.
2 options, 1 only add that concat to string values. 2 use quote() function.
In terms of code, i would prefer it to be double quoted `"` instead of `'` also because this will not need any code changes on my side.